packages/react/src/component-tests/IcDialog/IcDialog.cy.tsx

```diff
@@ -324,6 +325,26 @@ describe("IcDialog visual regression and a11y tests", () => {
     });
   });
 
+  it("should focus interactive content - search bar", () => {
+    mount(<DialogSearch />);
+
+    cy.get(DIALOG).should("exist");
+    cy.get("ic-button#slotted-dialog-btn").click();
+    cy.get(DIALOG).should(HAVE_ATTR, "open");
+    cy.wait(300);
```

Comment:
could you make a variable based on the number?

Reply:
It looks like the file is littered with various length waits. We have a ticket raised to improve the cypress tests generally (to better match cypress testing best practice) so will cover in that if ok?